crates/biome_css_analyze/src/lint/nursery/no_value_at_rule.rs

```diff
@@ -0,0 +1,72 @@
+use biome_analyze::{context::RuleContext, declare_rule, Ast, Rule, RuleDiagnostic};
+use biome_console::markup;
+use biome_css_syntax::CssAtRule;
+use biome_rowan::AstNode;
+
+declare_rule! {
+    /// Disallow use of @value rule in css modules
+    ///
+    /// Use of CSS variables is recommeneded instead of @value rule
+    ///
+    /// ## Examples
+    ///
+    /// ### Invalid
+    ///
+    /// ```css,expect_diagnostic
+    /// @value red: #FF0000
+    /// ```
+    ///
+    /// ### Valid
+    ///
+    /// ```css
+    /// :root {
+    ///   --red: #FF0000
+    /// }
+    ///
+    /// p {
+    ///   background-color: var(--red);
+    /// }
+    /// ```
+    ///
+    pub NoValueAtRule {
+        version: "1.8.0",
+        name: "noValueAtRule",
+        language: "css",
+        recommended: false,
+    }
+}
+
+impl Rule for NoValueAtRule {
+    type Query = Ast<CssAtRule>;
+    type State = CssAtRule;
+    type Signals = Option<Self::State>;
+    type Options = ();
+
+    fn run(ctx: &RuleContext<Self>) -> Option<Self::State> {
+        let node = ctx.query();
+
+        if node.rule().ok()?.as_css_value_at_rule().is_some() {
+            return Some(node.clone());
+        }
+
+        None
+    }
+
+    fn diagnostic(_: &RuleContext<Self>, node: &Self::State) -> Option<RuleDiagnostic> {
+        let span = node.range();
+        Some(
+            RuleDiagnostic::new(
+                rule_category!(),
+                span,
+                markup! {
+                    "Unexpected @value rule"
```

Comment:
These kinds of messages aren't helpful, because they don't provide good value to the user. Please check our rule pillars.

Maybe something like "Disallowed use of value detected" sounds better